articles/azure-arc/vmware-vsphere/recover-from-resource-bridge-deletion.md

```diff
@@ -11,23 +11,17 @@ ms.author: v-gjeronika
 manager: jsuri
 ---
 
-# Recover from accidental deletion of resource bridge VM
+# Recover from accidental deletion of resource bridge VM or it's irrecoverable
 
 In this article, you learn how to recover the Azure Arc resource bridge connection into a working state in disaster scenarios such as accidental deletion. In such cases, the connection between on-premises infrastructure and Azure is lost and any operations performed through Arc fail.
 
-## Prerequisites
-
-1. The disaster recovery script must be run from the same folder where the config (.yaml) files are present. The config files are present on the machine used to run the script to deploy Arc resource bridge. 
-
-1. The machine being used to run the script must have bidirectional connectivity to the Arc resource bridge VM on port 6443 (Kubernetes API server) and 22 (SSH), and outbound connectivity to the Arc resource bridge VM on port 443 (HTTPS).
-
-## Recovering the Arc resource bridge if there is VM deletion
+## Recovering the Arc resource bridge if there is VM deletion or it's irrecoverable
 
 To recover from Arc resource bridge VM deletion, you need to deploy a new resource bridge with the same resource ID as the current resource bridge using the following steps.
 
 1. Copy the Azure region and resource IDs of the Arc resource bridge, custom location, and vCenter Azure resources.
 
-2. Find and delete the old Arc resource bridge template from your vCenter.
+2. If the original yaml files are not accessible, **manually delete** the resource bridge VM from the vCenter (if it exists), and the resource bridge from azure. If the original YAML files are accessible, please run the script from the same folder where these files are present, and the script will handle the deletion of the ARB.
```

Comment:
@nascarsayan : Some observations for point#2:
1. Some context is required here for yaml files as the prerequisites section has been deleted.
2. Is **manually delete** part of the UI? If not, then we can have it as normal text. 
3. Please replace azure with Azure.
4. Please maintain consistency in the casing of yaml.
5. Delete please from the second sentence.
6. Please use expanded form of ARB.

Reply:
@riva-yadav please follow up here

Reply:
@BanoFarha , have updated, please take a look.